tools/pipelines/templates/build-npm-package.yml

```diff
@@ -420,6 +429,16 @@ stages:
               targetPath: '$(Build.ArtifactStagingDirectory)/test-files'
               artifactName: 'test-files'
               publishLocation: 'pipeline'
+        
+          # Point types back to default
+          - ${{ if or(eq(variables.releaseAlphaBetaBuild, 'alpha'), eq(variables.releaseAlphaBetaBuild, 'beta')) }}:
+            - task: Bash@3
+              displayName: Revert types to default
```

Comment:
Why do the types need to be reverted? Also, how do you know what "default" is for a package?